rtdata/languages/default

```diff
@@ -3297,7 +3297,7 @@ TP_LOCALLAB_GHS_LC_FRAME;Stretch Regularization & Midtones
 TP_LOCALLAB_GHS_LC_TOOLTIP;Slightly increases local contrast that was weakened by stretching.\nCan be completed by other tools specific to local contrast such as wavelets.
 TP_LOCALLAB_GHS_MID;Midtones
 TP_LOCALLAB_GHS_MID_TOOLTIP;Adjusts midtone balance after GHS transformation.\n Can be used to rebalance the midtones of the image after stretching.
-TP_LOCALLAB_GHS_BPFRAME_TOOLTIP;Sets the Black point (BP linear), White point (WP linear) and Highlight for a linear stretch of the image.\n\n * Black Point (BP linear) and White Point (WP linear) settings are only available if Stretch factor (D) is between 0.001 and 0.002 so as to not affect the histogram.\n\n * Black point (BP linear) and White point (WP linear) settings are sensitive to the general settings upstream of GHS: Highlight reconstruction, White balance and RAW processes.\n\n * To have relevant Black Point (BP linear) and White Point (WP linear) settings, the entire image must be analyzed. It is recommended to use 'Fit whole image to screen - Shortcut Alt-f'.
+TP_LOCALLAB_GHS_BPFRAME_TOOLTIP;Sets the Black point (BP linear), White point (WP linear) and Highlight for a linear stretch of the image.\n\n * Black Point (BP linear) and White Point (WP linear) settings are only available if Stretch factor (D) is between 0.001 and 0.002 so as to not affect the histogram.\n\n * When Black point and White point settings are accessible (when Stretch factor is between 0.001 and 0.002), all settings are neutralized (have a negligible effect) - Local Intensity (b), Symmetry point (SP), Protect shadows (LP), Protect highlights (HP), or are disabled : 1) Stretch Regularisation and Midtones: Value (LC), Midtones; 2) Highlight attenuation.\n\n * Black point (BP linear) and White point (WP linear) settings are sensitive to the general settings upstream of GHS: Highlight reconstruction, White balance and RAW processes.\n\n * To have relevant Black Point (BP linear) and White Point (WP linear) settings, the entire image must be analyzed. It is recommended to use 'Fit whole image to screen - Shortcut Alt-f'.
```

Comment:
The comment about the settings being "neutralized" does not directly relate to the black and white points. It is directly related to the stretch factor, so the comment should be in the stretch factor's tooltip.

My suggestion for the wording:
```
When Stretch factor is 0.002 or less, all stretch settings except for Black point and White point have no effect.
```